kernelci.org/config.toml

```diff
@@ -85,8 +85,3 @@ footer_about_disable = false
     identifier = "lf"
     name = "LF"
     url = "https://foundation.kernelci.org"
-
-  [[menu.main]]
```

Comment:
I think we should keep the Blog menu entry, just drop the `url` as it'll be using the standard one from the Docsy theme.  See this example website which features a blog section:
https://example.docsy.dev/

Reply:
The current implementation would result in two headings, "News" and "Posts".

![image](https://user-images.githubusercontent.com/24474678/132866420-a9b25b05-82b7-460b-973b-7d27569937fb.png)

Happy to merge into a single "Blog" entry if you'd prefer that.

Reply:
I'll merge the two categories into one.

Reply:
I've kept the two categories, but changed it so we have a single "Blog" link on the main menu